keyboards/studiokestra/galatea/rev3/config.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 studiokestra (@studiokestra)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define WS2812_PIO_USE_PIO1
```

Comment:
You arent using any other pio based driver so this isnt required.

With this, the whole file should be removed.